integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/VariantAnimationTimelineTest.kt

```diff
@@ -15,203 +15,85 @@
+ * tNum).roundToInt() AnimType.DifferentDurations -> 0 } SmartAnimateTransition(
+ * tween(durationMillis = durationMillis, easing = defaultEasing), delayMillis, ) } else {
+ * SmartAnimateTransition( tween( durationMillis = (1f * 1000.0).roundToInt(), easing =
+ * defaultEasing, ), 0, ) } } ) ) { VariantAnimationTimelineTestDoc.Main(sceneState =
+ * sceneState.value) }
+ *
+ * Column(Modifier.absoluteOffset(y = 750.dp)) { Row( horizontalArrangement =
+ * Arrangement.spacedBy(10.dp), verticalAlignment = Alignment.CenterVertically, ) { Text("Animation
+ * Type:", fontSize = 30.sp, color = Color.Black)
+ * com.android.designcompose.testapp.validation.TestButton( name = "Normal", tag = "Normal",
+ * selected = animType.value == AnimType.Normal, ) { animType.value = AnimType.Normal }
+ * com.android.designcompose.testapp.validation.TestButton( name = "OneByOne", tag = "OneByOne",
+ * selected = animType.value == AnimType.OneByOne, ) { animType.value = AnimType.OneByOne }
+ * com.android.designcompose.testapp.validation.TestButton( name = "DifferentDurations", tag =
+ * "DifferentDurations", selected = animType.value == AnimType.DifferentDurations, ) {
+ * animType.value = AnimType.DifferentDurations } } Row( horizontalArrangement =
+ * Arrangement.spacedBy(10.dp), verticalAlignment = Alignment.CenterVertically, ) { Text("Child
+ * Animation Type:", fontSize = 30.sp, color = Color.Black)
+ * com.android.designcompose.testapp.validation.TestButton( name = "Normal", tag = "ChildNormal",
+ * selected = childAnimType.value == ChildAnimType.Normal, ) { childAnimType.value =
+ * ChildAnimType.Normal } com.android.designcompose.testapp.validation.TestButton( name = "Early",
+ * tag = "ChildEarly", selected = childAnimType.value == ChildAnimType.Early, ) {
+ * childAnimType.value = ChildAnimType.Early }
+ * com.android.designcompose.testapp.validation.TestButton( name = "Late", tag = "ChildLate",
+ * selected = childAnimType.value == ChildAnimType.Late, ) { childAnimType.value =
+ * ChildAnimType.Late } } Row( horizontalArrangement = Arrangement.spacedBy(10.dp),
+ * verticalAlignment = Alignment.CenterVertically, ) { Text("Variant:", fontSize = 30.sp, color =
+ * Color.Black) com.android.designcompose.testapp.validation.TestButton( name = "Columns", tag =
+ * "Columns", selected = sceneState.value == SceneState.Closed, ) { sceneState.value =
+ * SceneState.Closed } com.android.designcompose.testapp.validation.TestButton( name = "Spread", tag
+ * = "Spread", selected = sceneState.value == SceneState.BeforeWelcome, ) { sceneState.value =
+ * SceneState.BeforeWelcome } com.android.designcompose.testapp.validation.TestButton( name =
+ * "Center", tag = "Center", selected = sceneState.value == SceneState.CenterAll, ) {
+ * sceneState.value = SceneState.CenterAll }
+ * com.android.designcompose.testapp.validation.TestButton( name = "Cluster", tag = "Cluster",
+ * selected = sceneState.value == SceneState.Open, ) { sceneState.value = SceneState.Open } } } }
+ */
```

Comment:
What happened here? Can you revert this?

Reply:
That would be ktfmt... hm, that's not great, let me try to figure out something

Reply:
I renamed the files to `<filename>.DISABLED` so that ktfmt wouldn't run on them.

Reply:
Ok as long as that doesn't lose the history. I'm surprised ktfmt does that to stuff in comments -- I don't remember that happening before.